utils/credentials/credentials.go

```diff
@@ -125,7 +125,7 @@ func ClientCredentials(server string) []grpc.DialOption {
 		if *insecure {
 			tlsConfig.InsecureSkipVerify = true
 		} else {
-			certificates, certPool, _ := LoadCertificates()
+			certificates, certPool := LoadCertificates()
```

Comment:
wasn't this fixed at: https://github.com/google/gnxi/pull/136/files ?

Reply:
Yeah I think I branched from the PR so it was included in the diff here